src/main/webapp/definitions/alfresco-model.yaml

```diff
@@ -0,0 +1,318 @@
+          description: Authentication failed
+        '404':
+          description: |
+            **aspectId** does not exist
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+
+definitions:
+  Error:
+    type: object
+    required:
+      - error
+    properties:
+      error:
+        type: object
+        required:
+          - statusCode
+          - briefSummary
+          - stackTrace
+          - descriptionURL
+        properties:
+          errorKey:
+            type: string
+          statusCode:
+            type: integer
+            format: int32
+          briefSummary:
+            type: string
+          stackTrace:
+            type: string
+          descriptionURL:
+            type: string
+          logId:
+            type: string
+  Pagination:
+    type: object
+    properties:
+      count:
+        type: integer
+        format: int64
+        description: |
+          The number of objects in the entries array.
+      hasMoreItems:
+        type: boolean
+        description: |
+          A boolean value which is **true** if there are more entities in the collection
+          beyond those in this response. A true value means a request with a larger value
+          for the **skipCount** or the **maxItems** parameter will return more entities.
+      totalItems:
+        type: integer
+        format: int64
+        description: |
+          An integer describing the total number of entities in the collection.
+          The API might not be able to determine this value,
+          in which case this property will not be present.
+      skipCount:
+        type: integer
+        format: int64
+        description: |
+          An integer describing how many entities exist in the collection before
+          those included in this list. If there was no **skipCount** parameter then the
+          default value is 0.
+      maxItems:
+        type: integer
+        format: int64
+        description: |
+          The value of the **maxItems** parameter used to generate this list.
+          If there was no **maxItems** parameter then the default value is 100.
+  AspectPaging:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/AspectEntry'
+  AspectEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Aspect'
+  Aspect:
+    type: object
+    required:
+      - id
+      - title
+    properties:
+      id:
+        type: string
+      title:
+        type: string
+      description:
+        type: string
+      parentId:
+        type: string
+      properties:
+        type: array
+        items:
+          $ref: '#/definitions/NodeDefinitionProperty'
+  NodeDefinitionProperty:
```

Comment:
I realise copy/paste is not ideal but can we re-use what is defined in alfresco-core.yaml exactly as we may one day combine these definitions and we don't want to have multiple definitions for the same thing.

NodeDefinitionProperty -> Property